internal/abi/fastly/hostcalls_noguest.go

```diff
@@ -297,6 +297,10 @@ func OpenDictionary(name string) (*Dictionary, error) {
 	return nil, fmt.Errorf("not implemented")
 }
 
+func (d *Dictionary) GetBytes(key string) ([]byte, error) {
+	return []byte{}, fmt.Errorf("not implemented")
```

Comment:
Why not `return nil, ...`?

Second, `fmt.Errorf("const string")` should be replaced with a declared `errors.New("...")` in a global or a custom `type errorString string` and a const.

Reply:
This is all over the stubbed implementations without `wasip1` build tags.

Reply:
Sure, I can return `nil`.  (The rest here I just copied what was already there. I didn't want to make changes that weren't related to the change in this PR.)

Reply:
(Patch to return `nil` pushed.)